modules/nf-core/glimpse2/concordance/main.nf

```diff
@@ -0,0 +1,55 @@
+process GLIMPSE2_CONCORDANCE {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::glimpse-bio=2.0.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/glimpse-bio:2.0.0--hf340a29_0':
+        'quay.io/biocontainers/glimpse-bio:2.0.0--hf340a29_0' }"
+
+    input:
+    tuple val(meta), path(estimate), path(estimate_index), path(truth), path(truth_index), path(freq), path(freq_index), path(samples), val(region)
+    tuple val(meta2), path(groups), val(bins), val(ac_bins), val(allele_counts)
+    val(min_val_gl)
+    val(min_val_dp)
+
+    output:
+    tuple val(meta), path("*.error.*.txt.gz") , emit: errors
+    tuple val(meta), path("*.rsquare.*.txt.gz"), emit: rsquare
```

Comment:
Can you make a separate output for each `errors` or `rsquare` file created? It's usually easier to merge them than to split them on file name for example :)